src/app/organization/tellers/create-teller/create-teller.component.ts

```diff
@@ -0,0 +1,86 @@
+/** Angular Imports */
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { ActivatedRoute, Router } from '@angular/router';
+import { DatePipe } from '@angular/common';
+
+/** Custom Services */
+import { OrganizationService } from 'app/organization/organization.service';
+
+/**
+ * Create Create Teller component.
+ */
+@Component({
+  selector: 'mifosx-create-teller',
+  templateUrl: './create-teller.component.html',
+  styleUrls: ['./create-teller.component.scss']
+})
+export class CreateTellerComponent implements OnInit {
+
+  /** Teller Form */
+  tellerForm: FormGroup;
+  /** Office data. */
+  officeData: any;
+   /** Minimum Date allowed. */
```

Comment:
Fix indentation here### 
```suggestion
  /** Minimum Date allowed. */
```